app/src/main/java/com/kickstarter/libs/TrackingClientType.java

```diff
@@ -22,12 +24,38 @@ public final void track(final String eventName) {
 
     final boolean userIsLoggedIn = loggedInUser() != null;
 
-    hashMap.put("time", time());
-    hashMap.put("user_logged_in", userIsLoggedIn);
+    hashMap.putAll(sessionProperties(userIsLoggedIn));
 
     return hashMap;
   }
 
+  private @NonNull Map<String, Object> sessionProperties(final boolean userIsLoggedIn) {
+    final Map<String, Object> properties = new HashMap<String, Object>() {
+      {
+        put("android_uuid", androidUUID());
```

Comment:
On iOS we used to have `ios_uuid` but removed it since it's just the same value as `device_distinct_id`. You might be able to remove this prop as well.